run-make-check.sh

```diff
@@ -69,7 +69,13 @@ function run() {
     BUILD_MAKEOPTS=${BUILD_MAKEOPTS:-$DEFAULT_MAKEOPTS}
     CHECK_MAKEOPTS=${CHECK_MAKEOPTS:-$DEFAULT_MAKEOPTS}
 
-    $DRY_RUN ./do_cmake.sh -DWITH_FIO=ON $@ || return 1
+    # Do the right thing if Python 2 is absent - http://tracker.ceph.com/issues/23035
```

Comment:
this comment is not necessary, the code is self-documenting already.

Reply:
dropped the comment